<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QWidget" name="view">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>245</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbModel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRadio">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>48</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>采样数</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFrame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>86</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合成帧率</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbExpTime">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>曝光时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMsg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>202</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存成功</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
